server/src/Irony.cpp

```diff
@@ -346,8 +356,16 @@ void Irony::complete(const std::string &file,
         }
       }
 
+      // Raise a warning if the candidate may be susceptible to:
+      // https://github.com/Sarcasm/irony-mode/issues/367
+      if (availability == CXAvailability_NotAccessible) {
+        brief = "Irony: Warning: Candidate: `" +
+                prototype +
```

Comment:
Showing the prototype does not seem useful, we don't show it for the brief, the user is supposed to already be aware of the context.